pwndbg/commands/heap.py

```diff
@@ -79,7 +79,11 @@ def format_bin(bins, verbose=False, offset=None):
 
     return result
 
-@pwndbg.commands.ParsedCommand
+
+parser = argparse.ArgumentParser()
+parser.description = "Prints out chunks starting from the address specified by `addr`."
+parser.add_argument("addr", nargs="?", type=int, default=None, help="The address of the heap.")
+@pwndbg.commands.ArgparsedCommand(parser)
```

Comment:
We should delete the function docstring

Reply:
I wasn't sure if it was worth leaving them. If you think it makes sense, I will add a commit deleting all the doc strings.